src/components/rewind/rewind.js

```diff
@@ -44,7 +55,10 @@ class Rewind extends Component {
     const step = this.props.step || REWIND_DEFAULT_STEP;
     const from = this.props.player.currentTime;
     if (this.props.player.currentTime - step < 0) {
-      to = 0;
+      // In dvr when close to beginning dont rewind
+      if (!this.props.isDvr) {
```

Comment:
doesn't this mean that rewind won't take us all the way back? it will always remain a `REWIND_DEFAULT_STEP` from actual 0?

Reply:
It will some margin < REWIND_DEFAULT_STEP but this is better behaviour than move to the actual 0 which causes jumps of a whole segment sometimes